### PR TITLE
Notes continued

### PR DESCRIPTION
**Pull request by tarunJeevan**
Let me know what you guys think and if you have any questions.

### CODE REVIEW

File: js/notebook.js [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
-    a.append(document.createTextNode("New Notebook"));
+    // Return the <input>
+    return input;
+}
 
-    // Return the <a>
-    return a;
+function renameNotebook() {
+    // Toggle editable for element
+    console.log("dblclick works")
```
Comment (zeshengchen) [inline js/notebook.js:28]: If this is just for debugging, please remove it in the final code!

File: js/notebook.js [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+    return false;
+}
+
+function showNotebook() {
+    // Change notebook display using notes retrieved from database
+    console.log("onclick works")
```
Comment (zeshengchen) [inline js/notebook.js:34]: Remove it!

**Review by zeshengchen — APPROVED** [review]
See my comments

File: js/notebook.js (hunk 1 above)
Comment (tarunJeevan) [reply js/notebook.js:28]: Notebook.js is superficial to the current code. It's connected to a previous version I was working on, but I haven't decided if I want to implement both versions or just one, so I left it as it is. 
I'll remove the debugging portions once it's been merged though. I forgot I left them there.

**Review by kaylebredner — APPROVED** [review]

**Review by maitmd — APPROVED** [review]
Looks good from what I can tell

File: js/notesAPI.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+        } else { // Create new note
+
+            savedNote.id = Math.floor(Math.random() * 1000000) // TODO: Create anon function to check if id already exists in notes
+            savedNote.updated = new Date().toISOString()
+            notes.push(savedNote)
+        }
+
+        localStorage.setItem("notesapp-notes", JSON.stringify(notes)) // Save updated notes list to local storage
+    }
```
Comment (maitmd) [inline js/notesAPI.js:31]: Are you storing the notes to the database anywhere? Maybe I'm missing it?

Comment (tarunJeevan) [reply js/notesAPI.js:31]: We don't have a database yet, so I'm storing my notes in the browser's local storage for now. Once we have a database set up, I'll make my own collection Schema and store the notes in that

**Review by AlexBishop01 — APPROVED** [review]

**Pull request merged**